list-your-project.md

```diff
@@ -7,80 +7,51 @@ description: How you can list your TestNet project or DApp on the Moonbeam docum
  
 ![Template banner image](images/list-dapps-banner.png)
 
-**Disclaimer:** The content in this section is entirely managed by the projects themselves. Moonbeam is a permissionless network. Any project can deploy its contracts to Moonbeam. 
+## Introduction to State of the DApps
 
-## Minimum Content
+[State of the DApps](https://www.stateofthedapps.com/){target=_blank} was started in 2016 by Joris Bontje to provide a convenient listing of all of the smart contracts or decentralized applications deployed to Ethereum. It has grown tremendously since then, supporting multiple chains and showcasing a wealth of information. In their own words, *“Whether you are looking for new users, testers, concept feedback, partners, or investors, submitting a DApp to this definitive registry will help your project gain traction.”*
 
-In general, to be considered and added to this list, your project/DApp must meet the following requirements in terms of content:
+State of the DApps maintains a rank of projects based on active users, transaction volume, developer activity, and profile strength. Moonriver is live on State of the DApps and at the time of writing, both the 2nd and 7th top ranked projects across all ecosystems were built on Moonriver. Previously, a crowd-sourced DApps Listing was maintained [here](/learn/dapps-list/) as part of the Moonbeam docs site. That registry is deprecated and replaced by State of the DApps. 
 
- - Introduction (see template file)
- - Show working contracts and/or front-ends deployed or connected to the Moonbase Alpha TestNet
- - Explain to users how they can test or integrate your project/DApp
- - Link the GitHub repos of the code
- - Link to communication channels
+![Template banner image](images/stateofthedapps/stateofthedapps1.png)
```

Comment:
can you please use kebab case for the file names (ie. `state-of-the-dapps/...`) and update the image description